firebase.json

```diff
@@ -322,6 +322,11 @@
           "source": "/docs/api/tools{,/**}",
           "destination": "/docs/tools/",
           "type": "302"
+        },
+        {
+          "source": "/docs/dev/community-tutorials/the-graph-tutorial.html",
```

Comment:
not sure this change relates to the BE API docs

Reply:
Removed, must have showed up from earlier